CONTRIBUTING_DOCS.md

```diff
@@ -61,6 +63,8 @@ The submission of a PR kicks off a continuous integration process which includes
 
 However, you can also run this after submitting your PR and experiencing an `htmlproofer` failure from the Semaphore job.
 
+> **Note**: Sometimes `htmlproofer` may deliver errors that do not reproduce. If you get an error from `htmlproofer` that seems incorrect, rerun `htmlproofer`. In addition, sometimes the Semaphore job will return spurious `htmlproofer` errors. In this event, one of the maintainers can restart the job. 
```

Comment:
I'd rather we fix the root of this problem than just accept that the test is flaky. Let's remove this disclaimer and set aside some time for someone (could be me) to actually investigate and resolve this.